lib/lambda_ethereum_consensus/p2p/gossip/gossipsub.ex

```diff
@@ -27,6 +27,15 @@ defmodule LambdaEthereumConsensus.P2P.GossipSub do
       # {"sync_committee_0", Types.SyncCommitteeMessage}
     ]
 
+    # Add blob sidecar topics
+    # NOTE: there's one per blob index in Deneb (6 blobs per block)
+    topics =
+      topics ++
+        (0..5
```

Comment:
For single function pipes I usually recommend not using pipes. It makes it harder to read. I think this could be better:

```elixir
topics =
  topics ++
    Enum.map(0..5, fn i ->
      {"blob_sidecar_#{i}", Types.BlobSidecar, &Handler.handle_blob_sidecar(&1, i)}
    end)
```